transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -31,6 +34,9 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
   // format is: metadata/{entity_type}/{entity_id}_{version}.json,
   // but filename returns just {entity_id}_{version}.json, so that is what we deal with.
   val metadataPattern: Regex = "([^_]+)_(.+).json".r
+  // format is: {dir_path}/{file_id}_{file_version}_{file_name},
+  // but the dir_path seems to be optional
+  val fileDataPattern: Regex = "(.*\\/)?([^_^\\/]+)_([^_]+)_(.+)".r
```

Comment:
https://xkcd.com/208/

Reply:
lol that's great 😆 we're all just waiting our moments to shine...